Chore/docs on new oci images (#4683)
minvws/nl-kat-coordination: docs/source/installation-and-deployment/separate-boefje-workers.rst

    Options:
      -p, --plugins TEXT              A list of plugin ids to filter on.
      -l, --log-level [DEBUG|INFO|WARNING|ERROR]
                                      Log level
      --help                          Show this message and exit.

This long running process will automatically filter the scheduler on tasks for the ``OCI_IMAGE`` specified through its env.
Additionally, you could start the process with a filter on specific plugin ids, like so:

.. code:: shell

    $ docker run --network nl-kat-coordination_boefjes ghcr.io/minvws/openkat/nmap -p nmap-udp  # optional filter
    1970-17-28T15:26:16.333927 [info] Starting runtime
    1970-17-28T15:26:16.334173 [info] Configured BoefjeAPI [base_url=http://boefje:8000, outgoing_request_timeout=30, images=['ghcr.io/minvws/openkat/nmap:latest'], plugins=['nmap-udp']]
    1970-17-28T15:26:16.331262 [info] Created worker pool for queue 'boefje'
    1970-17-28T15:26:16.334481 [info] Started listening for tasks from worker pid=16
    1970-17-28T15:26:16.334501 [info] Started listening for tasks from worker pid=17
    HTTP Request: POST http://boefje:8000/api/v0/scheduler/boefje/pop?limit=1 "HTTP/1.1 200 OK"

> Note that to start a container in "worker-mode", i
> t needs access to the network of the boefje API. This may have a

Reviewer: ```suggestion
Note that to start a container in "worker-mode", it
needs access to the network of the boefje API. This may have a
```
